### ISSUE TITLE
Bad version reported for Dell BMC

### ISSUE THREAD

**Issue by ofaurax** [packethost/packet-hardware#75]
On a Dell BMC iDrac 7.10.30, version reported is 7.10.00.

```
# ipmitool mc info
Device ID                 : 32
Device Revision           : 1
Firmware Revision         : 7.10
IPMI Version              : 2.0
Manufacturer ID           : 674
Manufacturer Name         : DELL Inc
Product ID                : 256 (0x0100)
Product Name              : Unknown (0x100)
.....
Aux Firmware Rev Info     : 
    0x00
    0x1e
    0x1e
    0x00
```

The code is there:
https://github.com/packethost/packet-hardware/blob/master/packethardware/utils.py#L535

So we can see that the `.30` is in fact in aux bytes 1 and 2, and in hex format `0x1e`.
In fact, it seems to be `.30.00` because reported version in iDrac is `7.10.30.00` so perhaps it's encoded in hex, bytes 2 and 3, and mirrored for similar use on little- and big-endian processors.

Original code assumed BCD format on byte 0.

I'm not sure how this should be handled. Perhaps version decoding should be handled differently according to Manufacturer ID.

**Comment by splaspood** [comment]
Although I guess that wouldn't work for the few dell hosts we have disabled host OS access to the BMC from...

**Comment by ofaurax** [comment]
Fixed by #76
